### PR TITLE
Fix AZP CI

### PR DESCRIPTION
**Pull request by ryjones**
Moved to AZP, no need for Travis support

### CODE REVIEW

**Comment by ryjones** [comment]
@danintel sure - I enabled it, but it looks like everything failed :)

**Comment by mikelodder7** [comment]
Good PR. Finally we can be off Travis once everything passes.

**Comment by mikelodder7** [comment]
I believe the reason these failed was due to not having the `SODIUM_BUILD_STATIC=1` environment variable.

**Comment by ryjones** [comment]
I added the variable, and everything but one passed:

> 2019-12-16T21:35:36.3348337Z    Compiling subtle v2.2.2
2019-12-16T21:35:36.3799477Z error[E0554]: `#![feature]` may not be used on the stable release channel
2019-12-16T21:35:36.3801130Z   --> /home/vsts/.cargo/registry/src/github.com-1ecc6299db9ec823/subtle-2.2.2/src/lib.rs:12:34
2019-12-16T21:35:36.3801504Z    |
2019-12-16T21:35:36.3801711Z 12 | #![cfg_attr(feature = "nightly", feature(asm))]
2019-12-16T21:35:36.3801929Z    |                                  ^^^^^^^^^^^^
2019-12-16T21:35:36.3802128Z 
2019-12-16T21:35:36.3824897Z error[E0554]: `#![feature]` may not be used on the stable release channel
2019-12-16T21:35:36.3825819Z   --> /home/vsts/.cargo/registry/src/github.com-1ecc6299db9ec823/subtle-2.2.2/src/lib.rs:13:34
2019-12-16T21:35:36.3826184Z    |
2019-12-16T21:35:36.3826426Z 13 | #![cfg_attr(feature = "nightly", feature(external_doc))]
2019-12-16T21:35:36.3826688Z    |                                  ^^^^^^^^^^^^^^^^^^^^^
2019-12-16T21:35:36.3826899Z 
2019-12-16T21:35:36.4516046Z error: aborting due to 2 previous errors
2019-12-16T21:35:36.4516615Z 
2019-12-16T21:35:36.4522777Z For more information about this error, try `rustc --explain E0554`.

**Comment by mikelodder7** [comment]
The failing test needs to use the nightly rust compiler instead of stable. Change the install rust line for this test to be `curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly`

**Comment by ryjones** [comment]
@mikelodder7 take a look at line 74; that is what I use to install rust

File: azure-pipelines.yml [hunk 1]
```diff
@@ -69,7 +72,11 @@ jobs:
     steps:
       - script: |
           curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly
-        displayName: 'Install rust'
+          rustc --version
```
Comment (ryjones) [inline azure-pipelines.yml]: @mikelodder7 this is how I worked around https://github.com/rust-lang/rustup/issues/1079

**Comment by ryjones** [comment]
> The failing test needs to use the nightly rust compiler instead of stable. Change the install rust line for this test to be `curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly`

[Check this out](https://github.com/rust-lang/rustup/issues/1079). [Worked around it here](https://github.com/hyperledger/ursa/pull/87/files#r360011110).

File: azure-pipelines.yml [hunk 2]
```diff
@@ -69,7 +72,11 @@ jobs:
...
+          rustup toolchain install nightly
+          rustup default nightly
+          rustc --version
+        displayName: 'Install rust, select nightly'
```
Comment (ryjones) [inline azure-pipelines.yml]: @mikelodder7 this is how I worked around https://github.com/rust-lang/rustup/issues/1079

**Comment by ryjones** [comment]
[Proof this works once merged.](https://dev.azure.com/Hyperledger/Ursa/_build/results?buildId=4091&view=results)

**Pull request merged**